python-package/lightgbm/basic.py

```diff
@@ -18,9 +18,23 @@
 import numpy as np
 import scipy.sparse
 
-from .compat import (PANDAS_INSTALLED, PYARROW_INSTALLED, arrow_cffi, arrow_is_floating, arrow_is_integer, concat,
-                     dt_DataTable, pa_Array, pa_chunked_array, pa_ChunkedArray, pa_compute, pa_Table,
-                     pd_CategoricalDtype, pd_DataFrame, pd_Series)
+from .compat import (
```

Comment:
I love this format, it was super hard to get this right when inserting something in the middle with the previous one.

Reply:
same! I'm really happy with this one 😁